yt_dlp/extractor/hqporner.py

```diff
@@ -0,0 +1,63 @@
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    get_element_by_class,
+    parse_duration,
+    parse_resolution,
+    url_or_none,
+)
+
+
+class HqpornerIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?hqporner\.com/hdporn/(?P<id>[0-9]+)-(?P<display_id>[^.]+)\.html'
+    _TESTS = [{
+        'url': 'https://hqporner.com/hdporn/86482-all_night_rager.html',
+        'md5': '69cd373eb38aa82209c0450e0b9fc730',
+        'info_dict': {
+            'id': '86482',
+            'title': 'all night rager',
+            'display_id': 'all_night_rager',
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'ext': 'mp4',
+            'age_limit': 18,
+            'duration': 2434.0,
+            'tags': 'count:13'
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        iframe_url = self._html_search_regex(r'<iframe[^>]*src=[\'"]([^\'"]+)[^>]*allowfullscreen',
+                                             webpage, 'iframe url')
+
+        iframe_webpage = self._download_webpage(iframe_url, video_id, note='Downloading iframe webpage')
+
+        formats = []
+        for mobj in re.finditer(
```

Comment:
The data in the iframe page is a string containing HTML5: see https://github.com/ytdl-org/youtube-dl/blob/5966b430927997811a99e8b8c7421d37b46045cd/youtube_dl/extractor/hqporner.py#L68